### PR TITLE
[I18N] l10n_ar: add missing translations

### PR DESCRIPTION
**Pull request by bruno-zanotti**
### Task 392
Description of the issue/feature this PR addresses:
Add missing translations from task 366 in l10n_ar project. 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ar/i18n/es.po [hunk 1]
```diff
@@ -1263,9 +1352,21 @@ msgstr "Con Documento"
 #: code:addons/l10n_ar/models/account_move.py:0
 #, python-format
 msgid ""
-"You are trying to create an invoice for %s but you dont have an %s journal"
+"You are trying to create an invoice for domestic partner but you don't have "
+"an domestic market journal"
```
[the remark this answers is not in the document]
Comment (Bruno-Zanotti) [reply addons/l10n_ar/i18n/es.po]: @ren-odoo done!

**Comment by ren-odoo** [comment]
Hi @Bruno-Zanotti 

Thank you for the update, it looks ok now,  @jco-odoo this PR is ready for your review.

**Comment by mart-e** [comment]
Thanks, it looks good :+1: 
@robodoo r+

**Comment by jco-odoo** [comment]
Thanks @mart-e

**Pull request closed without merging**